### PR TITLE
MC-12824: Added edit custom field documentation

### PR DESCRIPTION
**Pull request by GuillaumeVialle**
#### Changes made
- Added documentation for edit custom fields.

#### Related PRs
- https://github.com/cloudops/cloudmc-core/pull/1084
- https://github.com/cloudops/cloudmc-ui/pull/1204

### CODE REVIEW

File: source/includes/administration/_custom_fields.md [hunk 1]
```diff
@@ -134,9 +134,55 @@ Required | &nbsp;
...
+
+Edit an existing custom field.
+
+Required | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The ID of the custom field.
+`field`<br/>*UUID* | The field you're defining, this is *not* the UI-facing name. It cannot be changed.
+`nameTranslations`<br/>*map* | Map of language short codes to name translations for the field. Translations for all languages are required.
+`type`<br/>*string* | The type of model for the field. It cannot be changed.
+`organization.id`<br/>*UUID* | The organization id that the custom fields are linked to. It cannot be changed.
```
Comment (fyrnaga) [inline source/includes/administration/_custom_fields.md:182]: not part of the example above

File: source/includes/administration/_custom_fields.md [hunk 2]
```diff
@@ -134,9 +134,55 @@ Required | &nbsp;
...
+	"type": "ORGANIZATION",
+	"field": "adminEmail",
+	"nameTranslations": {
+		"en": "Administrator's email",
+		"fr": "Courriel de l'administrateur"
+	},
+  "descriptionTranslations": {
+      "en": "username desc",
+      "fr": "username desc fr"
+  }
```
Comment (fyrnaga) [inline source/includes/administration/_custom_fields.md]: would be nice to have a proper indentation

File: source/includes/administration/_custom_fields.md [hunk 3]
```diff
@@ -134,9 +134,55 @@ Required | &nbsp;
...
+``` 
+
+Edit an existing custom field.
+
+Required | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The ID of the custom field.
+`field`<br/>*UUID* | The field you're defining, this is *not* the UI-facing name. It cannot be changed.
+`nameTranslations`<br/>*map* | Map of language short codes to name translations for the field. Translations for all languages are required.
+`type`<br/>*string* | The type of model for the field. It cannot be changed.
```
Comment (fyrnaga) [inline source/includes/administration/_custom_fields.md]: We should list the values as done in the create

File: source/includes/administration/_custom_fields.md (hunk 2 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_custom_fields.md]: My local was a mix of tabs and spaces so it looked good but not on Github. Changed that:

File: source/includes/administration/_custom_fields.md (hunk 1 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_custom_fields.md:182]: f8f56ef3b7ec1317459dbd9bfccffb093f9d00b1

File: source/includes/administration/_custom_fields.md (hunk 2 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_custom_fields.md]: f8f56ef3b7ec1317459dbd9bfccffb093f9d00b1

File: source/includes/administration/_custom_fields.md (hunk 3 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_custom_fields.md]: f8f56ef3b7ec1317459dbd9bfccffb093f9d00b1

**Review by fyrnaga — APPROVED** [review]
lgtm

**Review by evanluc — APPROVED** [review]

**Pull request merged**